src/esp/sim/ReplayBatchRenderer.cpp

```diff
@@ -29,15 +29,18 @@ ReplayBatchRenderer::ReplayBatchRenderer(
   sceneManager_ = scene::SceneManager::create_unique();
 
   for (int envIdx = 0; envIdx < config_.numEnvironments; ++envIdx) {
-    auto assetCallback =
+    auto callbacks = gfx::replay::createSceneGraphPlayerCallbacks();
```

Comment:
@eundersander Not sure if virtual methods would make sense here. Some of the callbacks require capturing local scope which would be difficult to do with virtual methods. I would venture that this is the reason they were callbacks before.